docs/usage/oauth2.md

```diff
@@ -23,6 +24,7 @@ ui.initOAuth({
     appName: "your-app-name",
     scopeSeparator: " ",
     additionalQueryStringParams: {test: "hello"},
+    additionalFormParams: {audience: "me"}
```

Comment:
I think you are missing a trailing comma here

Reply:
Nice catch!